### PR TITLE
Pull Request for Popup Window for desktop and Mobile

### PR DESCRIPTION
**Pull request by jkanyi-web**
- Added files and images for popup windows on both desktop and mobile
- Updated the  HTML file with a popup window
- Update styles.css file with styling for a popup window
- Add popup window in separate modal.js file for neatness
- Fixed Linter errors

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Hi @jkanyi-web , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pair programming project but I can only see one participant. Your partner should contribute with at least one commit for your project to be valid

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by jkanyi-web** [comment]
Thanks, Osoro.

File: modal.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+            </button>
+            <button type="button" id="card-button-source" class="card-button">
+              See source
+              <i class="fab fa-github"></i>
+            </button>
+          </div>
+        </div>
+      </div>
+
+    </div>
```
Comment (leonardodiasb) [inline modal.js:130]: - [ ] Your modal is good and working fine. But on the desktop view it is possible to improve some CSS styling to make it more similar to the Figma template. Kindly add some margin below the popup window to make it centralized on the user's screen, replace the 'X' button to make it on the top right corner of the popup and please try to make the technologies tags centralized on its boxes just like the Figma template. Kindly make those changes to make your popup more professional and similar to the Figma template.
 - Your design:
 - 
image

 - Figma template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @jkanyi-web, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Linters passing. ✔️
 - Professional README and PR. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @jkanyi-web,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good commit messages ✔️
- Well organized and clean project ✔️

approval
# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by jkanyi-web** [comment]
Awesome, thanks!

**Pull request merged**